queries.sql

```diff
@@ -24,4 +24,93 @@ SELECT * FROM animals WHERE name <> 'Gabumon';
 -- 8. Find all animals with a weight between 10.4kg and 17.3kg (inclusive)
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
 
+--- Task 2 ----
+UPDATE animals
+SET species = 'unspecified';
```

Comment:
- [ ] This is the line of code I am referring to: Please put it below `Begin Transaction` as a requirement.